spec.html

```diff
@@ -32162,26 +32223,122 @@ <h1>
         </emu-note>
       </emu-clause>
 
-      <emu-clause id="sec-defaulttimezone" type="implementation-defined abstract operation">
-        <h1>DefaultTimeZone ( ): a String</h1>
+      <emu-clause id="sec-time-zone-identifier-record">
+        <h1>Time Zone Identifier Record</h1>
+        <p>A <dfn variants="Time Zone Identifier Records">Time Zone Identifier Record</dfn> is a Record used to describe an available time zone identifier, as well as the primary time zone identifier corresponding to it.</p>
```

Comment:
Re the name, it might be a bit confusing to have both "time zone identifier" and "time zone identifier record" as distinct types, because the "Record" in a record schema name is occasionally elided (e.g. completion record -> completion, environment record -> environment), which would produce a collision in this case.

Maybe "Time Zone Identification Record"? Not sure.

Reply:
Good point. Maybe simplest just to call it "Time Zone Record" which doesn't conflict with anything else?

Reply:
Isn't "Time Zone" a thing in Temporal, though?

Reply:
Yeah, I guess that could be confused too. 

I'm not super-enthusiastic about "Time Zone Identification Record" because "identification" implies to me that additional info would be in the record besides just the identifier and its corresponding primary identifier.  Like "What's your name?" (what this record does) vs. "Show me your papers!" (what  "identification" seems to imply).

That said, I don't have a strong opinion on this, I'm happy to follow the crowd here.

If choosing between "Time Zone Record", "Time Zone Identifier Record", and "Time Zone Identification Record", do you have a preference?  Or is there some other name that would work better than those three?

Reply:
"Time Zone Identifier Record" seems most accurate to me. I'd rather ensure we don't elide "record" than pick a less accurate name. Let's see what the editors think.

Reply:
OK, for now I left it as "Time Zone Identifier Record" in my latest commit, but applied all of @jmdyck's other feedback.

Happy to change the record type's name later if that's what the editors prefer.

Reply:
Haven't seen any other concerns about this, so I'll resolve. Let me know if you want me to re-open. Thanks!